specifyweb/frontend/js_src/lib/components/FormParse/fields.ts

```diff
@@ -212,18 +212,23 @@ const processFieldType: {
       maxLength: f.parseInt(getProperty('maxLength')),
     };
   },
-  QueryComboBox({ getProperty, fields }) {
+  QueryComboBox({ getProperty, fields, table, cell }) {
     if (fields === undefined) {
       console.error('Trying to render a query combobox without a field name');
       return { type: 'Blank' };
     } else if (fields.at(-1)?.isRelationship === true) {
+      const isParentTaxon =
```

Comment:
why make this logic specific to a single table?
try to avoid introducing edge cases - they make testing harder as there are more cases needed to test to exercise all the code path (though in practice that more likely means that table-specific or collection-specific bugs could be hidden in the code for longer until some user discovers them)

instead, if the form definition explicitly says `editBtn=false`, but does not explicitly include `viewBtn`, then why not default viewBtn to true in that case?
that logic seems desirable in all tables, not just Taxon...